test/html-to-react-tests.js

```diff
@@ -101,6 +101,47 @@ describe('Html2React', function() {
 
             assert.equal(reactHtml, htmlExpected);
         });
+
+        it('should not insert spans into tables even if there is white space', function() {
```

Comment:
As far as I can tell, this test doesn't add anything except verify that whitespace text nodes are stripped.